lib/i18n_sonder/workers/upload_source_strings_worker.rb

```diff
@@ -27,7 +27,7 @@ def perform(object_type, object_id, translated_attribute_params)
         klass = object_type.constantize
         object = klass.find(object_id)
         if object.present?
-          updated_at = object.updated_at.to_i
+          updated_at = object.has_attribute?(:updated_at) ? object.updated_at.to_i : nil
```

Comment:
I believe ActiveRecord models have `updated_at` column by default. Is there an edge case?

Reply:
I didn't know that, but the exception that is causing me issues is `app/models/storage_location.rb` in Flatbook.